src/main/java/games/strategy/net/nio/NioReader.java

```diff
@@ -146,7 +146,7 @@ private void loop() {
       } catch (final Exception e) {
         // catch unhandles exceptions to that the reader
         // thread doesnt die
-        logger.log(Level.WARNING, "error in reader", e);
+        logger.log(Level.SEVERE, "error in reader", e);
```

Comment:
What about using logger.warn?

Reply:
We're still using the java.util api here and not a logback api regrettable. So there is no 'warn' method, there is a 'warrning' method, but it also does not take an exception as an arg so we woudl lose the stack trace.

Reply:
Meant warning, but you are right. The stacktrace is valueable

Reply:
Also it should be the severe method, but same problem...